examples/terraform-cloud-enterprise/README.md

```diff
@@ -36,10 +36,23 @@ jobs:
           INFRACOST_TERRAFORM_CLOUD_TOKEN: ${{ secrets.TFC_TOKEN }} 
           # INFRACOST_TERRAFORM_CLOUD_HOST: my-tfe-host.com # For Terraform Enterprise users only.
         
-      - name: Post the comment
-        uses: infracost/actions/comment@v1
-        with:
-          path: /tmp/infracost.json
-          behavior: update # Create a single comment and update it. See https://github.com/infracost/actions/tree/master/comment for other options
+      - name: Post Infracost comment
+        run: |
+
+          # Posts a comment to the PR using the 'update' behavior.
+          # This creates a single comment and updates it. The "quietest" option.
+          # The other valid behaviors are:
+          #   delete-and-new - Delete previous comments and create a new one.
+          #   hide-and-new - Minimize previous comments and create a new one.
+          #   new - Create a new cost estimate comment on every push.
+          #
+          # See https://www.infracost.io/docs/features/cli_commands/#comment-on-pull-requests
+          # for other inputs such as target-type.
+
```

Comment:
Can we remove the empty lines around this comment block in all examples? It seems bulky to me.

Reply:
fixed in https://github.com/infracost/actions/pull/57/commits/17e37dde5f62c22991798df0bf567348091ee3a5